### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by stev-glodowski**
minor updates

### CODE REVIEW

**Review by ddimatos — APPROVED** [review]
Thank you.

File: z_infra_provisioning/cloud_infra_center/ocp_upi/README.md [hunk 1]
```diff
@@ -1,28 +1,28 @@
 # IBM® Cloud Infrastructure Center
 
-IBM® Cloud Infrastructure Center is an advanced infrastructure management offering, including on-premises cloud deployments of IBM z/VM®-based and KVM based Linux® virtual machines on the IBM Z® and IBM LinuxONE platforms.
+IBM® Cloud Infrastructure Center is an advanced infrastructure management product, providing on-premises cloud deployments of IBM z/VM®-based and KVM based Linux® virtual machines on the IBM Z® and IBM LinuxONE platforms.
```
Comment (gyohuangxin) [inline z_infra_provisioning/cloud_infra_center/ocp_upi/README.md:3]: Thank you, Stev. However, I pick up this line from our official doc: https://www.ibm.com/docs/en/cic/1.1.3?topic=overview

**Comment by ddimatos** [comment]
I resolved your conflict it was minor. Generally its best the committer resolve conflicts to avoid any programmatic errors/assumptions.

File: z_infra_provisioning/cloud_infra_center/ocp_upi/README.md (hunk 1 above)
Comment (ddimatos) [reply z_infra_provisioning/cloud_infra_center/ocp_upi/README.md:3]: I don't know if this conversation was resolved, but a PR was made so I assume it was. You can also perform PRs between the your team.

**Pull request merged**